migrations/migration-9/index.js

```diff
@@ -112,16 +116,18 @@ async function pinsToDAG (blockstore, datastore, pinstore, onProgress) {
   }
 
   onProgress(100, 'Updating pin root')
-  const pinRoot = new dagpb.DAGNode(new Uint8Array(), [
-    await pinset.storeSet(blockstore, PinTypes.recursive, recursivePins),
-    await pinset.storeSet(blockstore, PinTypes.direct, directPins)
-  ])
-  const buf = pinRoot.serialize()
-  const cid = await dagpb.util.cid(buf, {
-    cidVersion: 0
-  })
+  const pinRoot = {
+    Links: [
+      await pinset.storeSet(blockstore, PinTypes.direct, directPins),
+      await pinset.storeSet(blockstore, PinTypes.recursive, recursivePins)
+    ]
+  }
+  const buf = dagPb.encode(pinRoot)
```

Comment:
`const block = Block.encode({ value: pinRoot, codec: dagPb, hasher: sha256 })` is available to shorten this slightly, then you can pick `block.cid.bytes` out of it.

Reply:
It looks like `Block.encode` will [only ever give me a v1 CID back](https://github.com/multiformats/js-multiformats/blob/master/src/block.js#L149) and we need a v0 CID here?

Reply:
Could just pull the multihash bytes out of the v1 CID, I guess..

Reply:
mmm, `block.cid.multihash` should do it. Alternatives include `block.cid.toV0().bytes` (which is more explicit but a little wasteful) or adding a `cidVersion` argument to `Block.encode`.

Reply:
I don't think Block.encode will save that much, and probably it's best to not add support for v0 there if we eventually want to break from v0. I would personally leave it as is.